components/woo/order.php

```diff
@@ -1398,7 +1403,7 @@ protected function sanitize_address( $data, $prefix = '' ) {
 			$data[ $prefix . 'zip_code' ] = $data[ $prefix . 'postcode' ];
 		}
 
-		return array_filter( $data );
+		return array_filter( $data, array(__CLASS__, 'filterArrayPreserveEmptyString') );
```

Comment:
`__CLASS__` should be `$this` in this context. Dunno why it works like that because it doesn't say `public static function filterArrayPreserveEmptyString` (missing `static`).

Reply:
👍